js/components/portal-dashboard/feedback/question-level-feedback-student-rows.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { Map } from "immutable";
+import Answer from "../../../containers/portal-dashboard/answer";
+import { getFormattedStudentName } from "../../../util/student-utils";
+import AwaitingFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/awaiting-feedback-question-badge-icon.svg";
+import GivenFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/given-feedback-question-badge-icon.svg";
+import UpdateFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/update-feedback-question-badge-icon.svg";
+
+import css from "../../../../css/portal-dashboard/feedback/feedback-rows.less";
+
+interface IProps {
+  activityId: string | null;
+  activityIndex: number;
+  answers: any;
+  currentQuestion: any;
+  feedbacks: Map<any, any>;
+  feedbacksNeedingReview: Map<any, any>;
+  isAnonymous: boolean;
+  students: Map<any, any>;
+  updateQuestionFeedback: (answerId: string, feedback: any) => void;
+}
+
+export const QuestionLevelFeedbackStudentRows: React.FC<IProps> = (props) => {
+  const { answers, currentQuestion, feedbacks, isAnonymous, students, updateQuestionFeedback } = props;
+
+  const handleFeedbackChange = (answerId: string) => (event: React.FormEvent<HTMLTextAreaElement>) => {
+    if (answerId) {
+      const target = event.currentTarget as HTMLTextAreaElement;
+      updateQuestionFeedback(answerId, {feedback: target.value});
+    }
+  };
+
+  const feedbackRows = students.map ((student: Map<any, any>, index: number) => {
+    const studentId = student.get("id");
+    const currentQuestionId = currentQuestion.get("id");
+    const formattedName = getFormattedStudentName(isAnonymous, student);
+    const answer = answers.getIn([currentQuestionId, studentId]);
+    const answerId = answer?.get("id");
+    const feedbackData = feedbacks.getIn([answerId]);
+    const feedback = feedbackData ? feedbackData.get("feedback") : "";
+
+    const awaitingFeedbackIcon = <AwaitingFeedbackQuestionBadgeIcon />;
+    const givenFeedbackIcon = <GivenFeedbackQuestionBadgeIcon />;
+    const updateFeedbackIcon = <UpdateFeedbackQuestionBadgeIcon />;
```

Comment:
Is this for future use? If so, we can comment out for now to eliminate the console warnings.